spec/features/user_resets_password_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+feature "user resets password" do
+  scenario "user resets password" do
+    
+    user = Fabricate(:user)
+    
+    visit sign_in_path
+    click_reset_password
+    assert_reset_password_page
+
+    enter_and_submit_email(user)
+    assert_confirmation_page
```

Comment:
Actually, I just looked at your template. I was thinking this would be in a flash message but I see I was wrong.